src/client/pages/Offer/Introduction/Sidebar/StartDate.tsx

```diff
@@ -456,11 +415,82 @@ export const StartDate = ({
 
       <CancellationOptions
         quotes={selectedQuotes}
-        loadingQuoteIds={loadingQuoteIds}
+        isDisabled={loadingQuoteIds.length > 0 || isLoading}
         onToggleCancellationOption={(isChecked, quoteId) =>
           handleSelectNewStartDate(isChecked ? null : new Date(), [quoteId])
         }
       />
     </>
   )
 }
+
+export const useStartDate = (): Omit<StartDateProps, 'size' | 'modal'> => {
```

Comment:
I would consider renaming the hook some something more explicit. Now it sounds like it would return a start date, while it's actually returning props.

Maybe something like `useStartDateProps`?